server/config/config.example.js

```diff
@@ -4,5 +4,13 @@
 module.exports = {
     db: {
         uri: '', //place the URI of your mongo database here.
-    }
+    },
+    mail: {
+        username: '',
+        key: '',
+        public_key: '',
+        from: '',
+        to: [''],
```

Comment:
We'll want to eventually integrate this into the user database so that users specify the address to send notifications to.